### DISCUSSION TITLE
CTKScrollableframe clear method does not work

### DISCUSSION THREAD

**Discussion by lordskyzw** [TomSchimansky/CustomTkinter#1775] [General]
for widget in self.scrollable_frame.winfo_children():
                  ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'CTkScrollableFrame' object has no attribute 'scrollable_frame'

**Comment by Awle** [comment]
This is not much to go by and understand your issue.
I guess you are trying to iterate over your CTkScrollableFrame (that you named scrollable_frame) widget's children?
Then you could try: 
```python
for widget in scrollable_frame.winfo_children():
```

**Comment by lordskyzw** [comment]
I actually fixed the issue, was wondering how to make a pr but this is the fix in the CTkScrollableFrame class:

def clear_frame(self):
        for widget in self.winfo_children():
            widget.destroy()

**Comment by Awle** [comment]
There is a destroy method for `CTkScrollableFrame`
It was not working?

**Comment by lordskyzw** [comment]
it destroys the whole frame, not the contents

**Comment by Awle** [comment]
OK there is no error then, you just wanted a method to clear the frame.
It was working as intended.
